recipes/libglu/bld.bat

```diff
@@ -0,0 +1,18 @@
+mkdir build
+cd build
+
+:: Configure.
```

Comment:
You are asking the wrong guy. I would skip it for now, get this into a feedstock, and then ask for help there. That way you get `Linux` and `OS X` out of the door and can start using it.